ibm/resource_ibm_tg_gateway_connection.go

```diff
@@ -74,6 +75,13 @@ func resourceIBMTransitGatewayConnection() *schema.Resource {
 				ForceNew:    true,
 				Description: "The ID of the network being connected via this connection. This field is required for some types, such as 'vpc'. For network type 'vpc' this is the CRN of the VPC to be connected. This field is required to be unspecified for network type 'classic'.",
 			},
+			tgNetworkAccountID: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Optional:    true,
```

Comment:
For non cross account is the account ID a computed fied...
Is this both optional and computeD?